apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,55 @@
+/************* GENERAL PAGE STYLES *************/
+
+html {
+  font-size: 18px;
+  font-family: 'Roboto', sans-serif;
+  color: rgb(73, 72, 72);
+}
+
+body {
+  margin: 0;
+  line-height: 1.7rem;
+}
+
+.container {
+  width: 80%;
+  margin: 0 auto;
+}
+
+/************* HEADER *************/
+
+header {
+  background: center / cover no-repeat url("/images/apple-pie.jpg");
+  height: 500px;
+}
+
+h1 {
+  color: white;
+  font-weight: 300;
+  font-size: 4rem;
+  text-align: center;
+  line-height: 500px;
+  vertical-align: center;
+  margin: 0;
```

Comment:
This declaration is part of the reset of the headings, maybe I would move this part at the top inside a reset section.